From: Takashi Iwai <tiwai@suse.de>
Subject: ALSA: hda: kabi workaround for generic parser flag
Patch-mainline: Never, kABI workaround
References: bsc#1051510

The patch ALSA-hda-realtek-Fix-overridden-device-specific-init.patch
introduced a new bit flag to hda_gen_spec which breaks kABI.  Add a
standard ifdef wrap over it.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 sound/pci/hda/hda_generic.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/sound/pci/hda/hda_generic.h
+++ b/sound/pci/hda/hda_generic.h
@@ -237,7 +237,9 @@ struct hda_gen_spec {
 	unsigned int indep_hp_enabled:1; /* independent HP enabled */
 	unsigned int have_aamix_ctl:1;
 	unsigned int hp_mic_jack_modes:1;
+#ifndef __GENKSYMS__ /* kABI workaround for SLE kernel */
 	unsigned int skip_verbs:1; /* don't apply verbs at snd_hda_gen_init() */
+#endif
 
 	/* additional mute flags (only effective with auto_mute_via_amp=1) */
 	u64 mute_bits;
